src/lib/stores/feedback.ts

```diff
@@ -110,11 +104,15 @@ function createFeedbackStore() {
                 return feedback;
             });
         },
+        // TODO: update growth server to accept `billingPlan` and other keys.
         submitFeedback: async (
             subject: string,
             message: string,
-            firstname?: string,
+            name?: string,
             email?: string,
+            // @ts-ignore
```

Comment:
Let's remove this so the build doesn't fail

Reply:
Done.